runelite-client/src/main/java/net/runelite/client/plugins/bank/BankPlugin.java

```diff
@@ -171,4 +181,57 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 
 		stringStack[stringStackSize - 1] += strCurrentTab;
 	}
+
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() != WidgetID.SEED_VAULT_GROUP_ID || !config.seedVaultValue())
+		{
+			return;
+		}
+
+		handleSeedVaultTotal();
+	}
+
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
```

Comment:
Is there a script to set the vault title like there is for banks?

Reply:
Per my understanding of a discussion with Polar and Abex this is created using a generic script that builds the layout of interfaces. 

> [5:56 PM] abex: just edit 631:1[1] whenever the inventory changes
[5:57 PM] abex: maybe onWidgetLoaded 631 too,

So i guess no?